lib/model/query/analytics.js

```diff
@@ -152,7 +152,7 @@ left join (
   where au."loggedAt" >= ${_cutoffDate}
   group by au."actorId"
 ) as activeUsers on activeUsers."actorId" = u."actorId"
-group by (p."id", r."id", r."name")`);
```

Comment:
I'm guessing that this worked before because `r.id` is the primary key. I doubt either `r.name` or `r.system` was strictly necessary here, as if you're grouping by `r.id`, you're automatically grouping by `r.id, r.system` (`r.system` doesn't do any additional grouping beyond what `r.id` already does). For example, this query works even though it groups by `projects.id`, not `projects.id, `projects.name`:

```sql
SELECT projects.id, projects.name, count(*) AS form_count
FROM projects
JOIN forms ON forms."projectId" = projects.id
GROUP by projects.id
LIMIT 5;
```

Maybe if `roles` is a view, `r.id` doesn't work in this same way?